README.md

```diff
@@ -65,9 +65,9 @@ To integrate VirgilCryptoWrapper into your Xcode project using CocoaPods, specif
 target '<Your Target Name>' do
   use_frameworks!
 
-  pod 'VirgilCryptoFoundation', '~> 0.15.2'
-  pod 'VirgilCryptoRatchet', '~> 0.15.2'
-  pod 'VirgilCryptoPythia', '~> 0.15.2'
+  pod 'VirgilCryptoFoundation', '~> 0.16.0-rc1'
+  pod 'VirgilCryptoRatchet', '~> 0.16.0-rc1'
+  pod 'VirgilCryptoPythia', '~> 0.16.0-rc1'
```

Comment:
For production release it should be change to '~> 0.16.0'

Reply:
Fixed